### DISCUSSION TITLE
How do I avoid cells rotation in placement?

### DISCUSSION THREAD

**Discussion by OuDret** [The-OpenROAD-Project/OpenROAD#4515] [Q&A]
I need a cell that does not rotate in both global and detailed placement. Is there any way to achieve it? I did not find any parameter.  

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/74424277/9d1b472c-dac6-4458-a3f6-f045e42abcfc)

Thanks

**Comment by maliberty** [comment]
If it is a standard cell the LEF for the cell gives its symmetry.

**Reply by OuDret** [reply]
I have tried removing SYMMETRY in cell LEF definition since i found in LEF/DEF Ref the following statement:
`Note: If you do not specify a SYMMETRY statement, only N orientation is tried.`

It did not work

**Reply by maliberty** [reply]
Then package and submit a test case.

**Reply by OuDret** [reply]
Here it is

In LEF files symmetry definition is removed.
To run:
`$ source counter_4Bbit.tcl`

[test_case.tar.gz](https://github.com/The-OpenROAD-Project/OpenROAD/files/13901818/test_case.tar.gz)

Thanks

**Reply by OuDret** [reply]
what would you say is happening?

**Reply by maliberty** [reply]
I think we are not handling symmetry correctly in this case as it is unusual.  I'll work on a fix next week as I'm OOO right now.

**Reply by OuDret** [reply]
Thanks for the effort! ^^

**Reply by OuDret** [reply]
Any news?

**Comment by maliberty** [comment]
As this isn't a usual use case it has been lower priority.  The simplest thing might be to just delete the rows that represent Y symmetry if you want to avoid such for your experiments.
